src/server.ts

```diff
@@ -14,6 +14,7 @@ const initializeServer = () => {
   server.put("/project/:projectId", project.editProjectById);
 
   // ticket
+  server.post("/projects/:projectId/columns/:columnId/tickets", ticket.createTicket);
```

Comment:
We may want to give projectId and columnId in the body. Let's discuss this later.